stable/admin/templates/statefulset.yaml

```diff
@@ -1,6 +1,10 @@
 apiVersion: "apps/v1"
 kind: StatefulSet
 metadata:
+  annotations:
+    description: |-
+      NuoAdmin statefulset resource for NuoDB domain administration layer.
```

Comment:
I am not certain what the NuoDB terminology is for the admin. It might be admin tier. @adriansuarez ?

Reply:
I'm happy to change that with more appropriate wording.

Reply:
I would just say "NuoDB Admin layer". Maybe @jleslie85 has an opinion once he is back.